sdk/communication/communication-chat/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/communication-chat",
-  "version": "1.4.0-beta.3",
+  "version": "1.5.0-beta.1",
```

Comment:
do we need to bump to 1.5.0 if 1.4.0 never GA'd?

Reply:
1.4.0 is about to GA. I think this PR just got a bit ahead of that.
The PR for 1.4.0 is here: https://github.com/Azure/azure-sdk-for-js/pull/27383

Reply:
1.4.0 is released yesterday